### ISSUE TITLE
MultiSelect component typing bug

### ISSUE THREAD

**Issue by Tanishq4331** [opengovsg/design-system#319]
I am trying to use the MultiSelect component with react. The selection of values from the default menu works. However, typing into the search field throws the following error:

```
_virtual_index_js__WEBPACK_IMPORTED_MODULE_1__.r.exports is not a function
TypeError: _virtual_index_js__WEBPACK_IMPORTED_MODULE_1__.r.exports is not a function
```

**To Reproduce**
Use the MultiSelectcomponent as below:

```      
<MultiSelect
   name="selection"
   placeholder={"Please select"}
   values={[]}
   items={[...Array(20).keys()].map((idx) => {
          return { label: `select me ${idx}`, value: `select me ${idx}` };
    })}
/>
```

**Desktop (please complete the following information):**
 - OS: MacOS Monterey 12.6.2
 - Browser: Chrome 112.0.5615.49
 - React version: 18.2.0

**Additional context**
If I initialize items to an empty array, an exception is no longer thrown when typing into the field.

**Comment by karrui** [comment]
Does this still happen with `alpha`? Should be fixed there. Also about to take `alpha` out of, well, alpha soon.

**Comment by karrui** [comment]
Closed as v1.2.0 has been released to latest, which solves this bug
